packages/web3torrent/src/constants.ts

```diff
@@ -1,9 +1,9 @@
 import {Status, Torrent} from './types';
 import {ChannelState} from './clients/payment-channel-client';
 import {bigNumberify} from 'ethers/utils';
-
+export const BLOCK_LENGTH = 1 << 14; // Standard request length. Setting less than this doesn't make sense.
```

Comment:
Looking into Webtorrent's, I saw that the chunk is always of a maximum of this length, so this looks like a good idea. Make the transfers always be a multiple of the maximum chunk length.
Most chunks are that size, except the last chunk of a torrent, and that one gets handled as an exception.

Reply:
How does a chunk or a block relate to a piece?

From below it looks like block = piece?

I don't see either term here https://en.wikipedia.org/wiki/Glossary_of_BitTorrent_terms

Reply:
A piece is composed of chunks. On small torrents (10mb or less, AFAIK), each piece has only one chunk. So in those torrents, the terms piece and chunks are pretty much the same thing.

On bigger torrents, pieces are split on smaller chunks, of a máximum of 16kib. That's what the offset in each request is for, to _tell_ the peer, which **part** of the piece is asking for (which chunk of the piece).

The issue that I found, is that as I lately tested this with small torrents, is that the jumpstart process didn't account for chunks. The details of the logic behind which piece is selected or ignored, is not... completely clear to me. I made a fix, that fixed that, and I'm trying to resolve all edge cases (for example, I'm now having issues, where multiple leechers end up paying more than they should, even with this fix).

Reply:
Looking into this, the model change is made at the 25mb threshold, it seems. Before that, 1 piece is equivalent to 1 chunk.